src/Microsoft.CodeQuality.Analyzers/VisualBasic/ApiDesignGuidelines/BasicEnumShouldNotHaveDuplicatedValues.vb

```diff
@@ -0,0 +1,63 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Analyzer.Utilities.Extensions
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines
+
+Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.ApiDesignGuidelines
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Public Class BasicEnumShouldNotHaveDuplicatedValues
```

Comment:
I am not really good with VB.NET so feel free to suggest better syntaxes for the whole class.